homeassistant/components/xiaomi_miio/strings.json

```
@@ -5,37 +5,37 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "incomplete_info": "Incomplete information to set up device, no host or token supplied.",
-      "not_xiaomi_miio": "Device is not (yet) supported by Xiaomi Miio.",
+      "not_xiaomi_miio": "Device is not (yet) supported by Xiaomi Home integration.",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "error": {
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "wrong_token": "Checksum error, wrong token",
       "unknown_device": "The device model is not known, not able to set up the device using config flow.",
-      "cloud_no_devices": "No devices found in this Xiaomi Miio cloud account.",
-      "cloud_credentials_incomplete": "Cloud credentials incomplete, please fill in username, password and country",
-      "cloud_login_error": "Could not log in to Xiaomi Miio Cloud, check the credentials."
+      "cloud_no_devices": "No devices found in this Xiaomi Home account.",
+      "cloud_credentials_incomplete": "Credentials incomplete, please fill in username, password and country",
```

As you changed "Cloud server country" to "Server region" this should be adapted, too:

```suggestion
      "cloud_credentials_incomplete": "Credentials incomplete, please fill in username, password and server region",
```